media-sound/clementine/clementine-1.2.3-r1.ebuild

```diff
@@ -0,0 +1,169 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=5
```

Comment:
1.2.3 will be removed sooner or later. IMHO no matter to enhance it beyond the bug fixing...